src/components/mes-signature-pad/mes-signature-pad.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, ViewChild, EventEmitter, Output } from '@angular/core';
+import { SignaturePad } from 'angular2-signaturepad/signature-pad';
+
+@Component({
+  selector: 'mes-signature-pad',
```

Comment:
Not 100% sure we should be prefixing components with Mes, it's a custom component in the app so it's linked it mes. I can see us having a load of components prefixed with Mes making everything a bit messy

Reply:
you are probably correct

Reply:
I agree on this one.

Reply:
renamed to signature-area